.github/workflows/rebuild_macos_arm64.yml

```diff
@@ -0,0 +1,31 @@
+name: Rebuild MacOS ARM and Push to PR
+on:
+  workflow_dispatch:
+
+# This workflow is triggered manually from the Actions tab.
+
+jobs:
+  rebuild:
+    runs-on: macos-13-xlarge
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install tooling
+        run: |
+          brew install make cmake gcc
+      - name: Go to the build directory
```

Comment:
nit: the name is slightly misleading. Something like Build would be better

Reply:
good eyes. that's a left-over from testing.